docs/swagger.json

```diff
@@ -170,7 +170,92 @@
                 }
             }
         },
-        "/api/mixmining/{pubkey}/history": {
+        "/api/mixmining/batch": {
```

Comment:
Routes including things like `batch` are not really a part of restful route design - a straight post to `/api/mixmining` is the right thing to do here.